src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -63,7 +63,7 @@ public function isProxyCandidate(Definition $definition)
     /**
      * {@inheritdoc}
      */
-    public function getProxyFactoryCode(Definition $definition, $id)
+    public function getProxyFactoryCode(Definition $definition, $id, $methodName = null)
```

Comment:
Maybe a cleaner solution would be to add a new method ``getMethodFor($id)`` in the compiled container. WDYT ?